packages/tooltip/src/tooltip.ts

```diff
@@ -53,6 +66,39 @@ export class Tooltip extends LitElement {
         this._variant = '';
     }
 
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.addEventListener('sp-overlay-query', this.onOverlyQuery);
```

Comment:
Possible over optimization/complexity incoming...

Is it reasonable that `sp-overlay-query` would be handled while `this.open===false` should we move to track `open` as a property and add/remove this listener on it's value change? In theory tooltips are closed more than they are open allowing for fewer listeners active at a time.

Reply:
Yah, I'm not sure the added complexity and potential bugs are worth it. I'd like to punt on this one.